deploy.sh

```diff
@@ -33,7 +33,7 @@ docker save cpmonitor:${env} -o cpmonitor.tar
 docker save klimaschutzmonitor-dbeaver:${env} -o klimaschutzmonitor-dbeaver.tar
 
 # Copy the images, the compose files, the certificate renewal cron job and the reverse proxy settings to the server
-scp -C -r cpmonitor.tar klimaschutzmonitor-dbeaver.tar docker-compose.yml crontab reload-cert.sh docker/reverseproxy/ monitoring@monitoring.localzero.net:/tmp/
+scp -C -r cpmonitor.tar klimaschutzmonitor-dbeaver.tar docker-compose.yml crontab reload-cert.sh backup.sh docker/reverseproxy/ monitoring@monitoring.localzero.net:/tmp/
```

Comment:
You're copying the script to /tmp on the server, but in deploy.sh and README.md you expect it to be in /home/monitoring, so you need to move it over after copying it.